### DISCUSSION TITLE
CustomValueComparator as Spring Bean

### DISCUSSION THREAD

**Discussion by szwowo** [javers/javers#1171] [Q&A] [answered]
Hello,
I didn't find any example in documentation of using CustomValueComparator as Spring Bean.
Is it an option?

I don't want to manually create `Javers` using `JaversBuilder`. I have implemented my own `ObjectIdTypeAdapter` with `@Component` annotation and it works fine. I am trying to do something simillar with `CustomValueComparator` but it's not working. Is there a way to register `CustomValueComparator` withtout adding it in `JaversBuilder` manually?

Thanks

**Comment by bartoszwalacik** [comment, marked as answer]
This option was added recently. See https://javers.org/release-notes#661

**Reply by szwowo** [reply]
Awesome, thanks!
